__tests__/app.errors.test.js

```diff
@@ -22,3 +22,23 @@ describe("404 General Not Found Error", () => {
       });
   });
 });
+
+describe("Errors for article id", () => {
```

Comment:
I think this should be alongside the articles tests personally and inside the same describe block